### ISSUE TITLE
Is it possible to use quantile regression for produce power law fit?

### ISSUE THREAD

**Issue by Ana-Bastos** [statsmodels/statsmodels#9159]
I would like to so a nonlinear quantile regression, using a power law fit.

**This is my current Code, where i can do a u-curved linear quantile regression like (Y = b o + b1X1 + b2X12)...**

df_org = pd.read_csv('Point1.csv',sep=';')
 
df=df_org.dropna()
 
x=df['Tp']
y=df['Hm0']
 
#Plot Scatter
fig = plt.figure()
fig.suptitle('Hm0 versus Tp')
z=density_scatter(x,y,bins=[10,10])
plt.scatter(x,y,c=z,cmap='viridis')
plt.colorbar()
 
 reg_exp = 'Hm0 ~ Tp + I(Tp**2)' #-For linear regression of second degree
 
#Carve out the y and X matrices
y_train, X_train = dmatrices(reg_exp, df, return_type='dataframe')
 
#Plot all regression lines for multiple quantiles
colors = ['orange', 'yellow', 'violet']
i=0
handles = []
quantiles = [0.05, 0.5, 0.95]
for q in quantiles:
    quantile_model = smf.quantreg(formula=reg_exp, data=df)
    quantile_model_results = quantile_model.fit(q=q)
    y_pred_quantile = quantile_model_results.predict(X_train)
 
    label=get_quantileLabel(q,quantile_model_results)
 
    quantile, = plt.plot(X_train['Tp'].sort_values(), y_pred_quantile.sort_values(),lw=1,zorder=2,
        color=colors[i], linestyle='--',  label=label)
    i = i+1
    handles.append(quantile)
 
plt.legend(handles=handles)
plt.show()

![357779ab-3945-42b9-bef4-ad1bd493f245](https://github.com/statsmodels/statsmodels/assets/51142185/20d652bf-1e83-40b9-ad32-759f5b0b09b5)

**I would like to change  the linear regression to a nonlinear regression ( Power (convex): Theta1 * X^Theta2)? Is it possible?**

**Comment by josef-pkt** [comment]
nonlinear functions are currently not supported by statsmodels.

To only workaround to get the parameters would be using profile likelihood.
fix theta2 for the quantile regression, estimate theta1 and get the value of the objective function `llf`?
Then do a grid search or scalar optimization for theta2 that has the lowest value of the objective function.

However, this will not provide inferential statistics, standard errors, confint, p-values, for theta1 and theta2.

Another alternative that would work in this case is to log-transform
`y = a * x^b` -> `log(y) = log(a) + b log(x)`

quantiles are preserved when using a monotonic transformation as in log
We can get the quantiles for the original units by just inverse transforming (exp) the results.
